### DISCUSSION TITLE
67内核升级好像有问题,patch打不上导致ax3600编译不过

### DISCUSSION THREAD

**Discussion by zcluo** [Boos4721/openwrt#15] [Bugs/问题反馈]
Applying /home/zcluo/openwrt-boos4721/target/linux/ipq807x/patches-5.10/118-arm64-dts-ipq8074-Fixup-PCIe-dts-nodes.patch using plaintext:
patching file arch/arm64/boot/dts/qcom/ipq8074.dtsi
Hunk #2 FAILED at 703.
Hunk #4 FAILED at 764.
2 out of 6 hunks FAILED -- saving rejects to file arch/arm64/boot/dts/qcom/ipq8074.dtsi.rej
Patch failed!  Please fix /home/zcluo/openwrt-boos4721/target/linux/ipq807x/patches-5.10/118-arm64-dts-ipq8074-Fixup-PCIe-dts-nodes.patch!
make[4]: *** [Makefile:32: /home/zcluo/openwrt-boos4721/build_dir/target-aarch64_cortex-a53_musl/linux-ipq807x_generic/linux-5.10.67/.prepared_2ed51f70b64deb68721d83491807dfb7] Error 1
make[4]: Leaving directory '/home/zcluo/openwrt-boos4721/target/linux/ipq807x'
make[3]: *** [Makefile:11: compile] Error 2
make[3]: Leaving directory '/home/zcluo/openwrt-boos4721/target/linux'
time: target/linux/compile#8.99#3.78#10.43
    ERROR: target/linux failed to build.

**Comment by Boos4721** [comment]
![](https://www.hualigs.cn/image/6147720eafc63.jpg)
**WORKS**
